src/store/request-password-reset/index.ts

```diff
@@ -0,0 +1,53 @@
+import { PayloadAction, createAction, createSlice } from '@reduxjs/toolkit';
+
+export enum SagaActionTypes {
+  RequestPasswordReset = 'requestPasswordReset',
+  EnterRequestPasswordResetPage = 'enterRequestPasswordResetPage',
+  LeaveRequestPasswordResetPage = 'leaveRequestPasswordResetPage',
+}
+
+export enum RequestPasswordResetStage {
+  SubmitEmail = 'submit-email',
+  Done = 'done',
+}
+
+export type RequestPasswordResetState = {
+  loading: boolean;
+  stage: RequestPasswordResetStage;
+  errors: string[];
+};
+
+export enum ResetPasswordErrors {
+  UNKNOWN_ERROR = 'UNKNOWN',
+  API_ERROR = 'API_ERROR',
```

Comment:
Does this error hold any meaning? I.E., are we reacting differently anywhere in the app? If not, we don't really need it. It's also a little misleading because the api returns specific errors and this is being used in the case where it's likely something other than the api failed.